Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1621,25 +1626,118 @@ def fitToViewport(self,Actor,vp,wc=None,geo=None):
           if flipX:
             cam.Azimuth(180.)
       return Renderer
-
-class VTKAnimate(animate_helper.AnimationController):
-   pass
-
-# class VTKAnimate(animate_helper.animate_obj):
-#   def __init__(self,*args,**kargs):
-#     animate_helper.animate_obj.__init__(self,*args,**kargs)
-#     self._initial_blink_done = False
-#   def draw2(self,frame):
-#     if self.create_flg == 1:
-#         self.current_frame = frame
-#         kargs = {}
-#         if self._initial_blink_done:
-#           kargs["noblink"]=True
-#         else:
-#           self._initial_blink_done = True
-#         self.vcs_self.backend.clear()
-#         self.vcs_self.put_png_on_canvas(self.animation_files[frame],
-#                 self.zoom_factor, self.vertical_factor, self.horizontal_factor,**kargs)
-#         if animate_helper.hasPyQt:
-#           self.signals.drew.emit()
-
+  def update_input(self,vtkobjects,array1,array2=None,update=True):
```

Comment:
It's a good idea to put a blank line in between function definitions for readability.